SqlScriptDom/Parser/TSql/TSql140.g

```diff
@@ -16151,6 +16151,15 @@ createColumnStoreIndexStatement [IToken tUnique, bool? isClustered] returns [Cre
             tOrder:Order
             (
                 identifierColumnList[vResult, vResult.OrderedColumns]
+                {
+					foreach (var col in vResult.OrderedColumns)
+					{
+						if (col.ColumnType == ColumnType.PseudoColumnGraphNodeId || col.ColumnType == ColumnType.PseudoColumnGraphEdgeId || col.ColumnType == ColumnType.PseudoColumnGraphFromId || col.ColumnType == ColumnType.PseudoColumnGraphToId)
```

Comment:
Should we have a helper function to check whether it is a graph column instead of listing them all out here?

Reply:
Good suggestion. Let me see if there's anything already in the code base. If not, I can look to add one.